homeassistant/components/netatmo/camera.py

```diff
@@ -275,6 +279,30 @@ def async_update_callback(self):
         self._is_local = camera.get("is_local")
         self.is_streaming = bool(self._status == "on")
 
+        if self._model == "NACamera":  # Smart Indoor Camera
+            self.hass.data[DOMAIN][DATA_EVENTS][self._id] = self.process_events(
+                self._data.events.get(self._id)
```

Comment:
since you use `get`, can this be `None` ?

Reply:
Yes, I'll make it return an empty dict.